src/world/climate.h

```diff
@@ -26,20 +26,22 @@ enum {
 	CLIMATE_COLD
 };
 
-enum{
+enum {
 	WEATHER_SUNNY,
 	WEATHER_PARTIALLY_CLOUDY,
 	WEATHER_CLOUDY,
 	WEATHER_RAIN,
 	WEATHER_HEAVY_RAIN,
-	WEATHER_THUNDER
+	WEATHER_THUNDER,
+	WEATHER_SNOW
```

Comment:
In vanilla RCT2 it just going back to its default weather = `WEATHER_PARTIALLY_CLOUDY`